DevAcademy/Services/PlacesService.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  PlacesService.swift
+//  DevAcademy
+//
+//  Created by Zdenko Čepan on 15/08/2023.
+//
+
+import Foundation
+
+
+protocol PlacesService {
+    func places() async throws -> Features
+}
+
+
+class ProductionPlacesService: PlacesService {
+    @MainActor
```

Comment:
Rekl bych, ze tady ten `@MainActor` jeste byt nemusi, protoze se hypoteticky muze klidne stat, ze ty `places()` bude pouzivat jeste nejaka dalsi Service nebo neco podobneho, ktere je nebude potrebovat na Main vlakne.
Tu anotaci `@MainActor` bych presunul az do PlacesObservableObject na metodu `loadData()`, kde uz jsme si jisti, ze tam uz to Main vlakno potrebujeme, protoze nam ta metoda upravi `features`, ktere sledujeme z View.